src/vscode-dts/vscode.proposed.terminalQuickFixProvider.d.ts

```diff
@@ -31,17 +31,21 @@ declare module 'vscode' {
 
 	export namespace window {
 		/**
-			 * @param provider A terminal quick fix provider
-			 * @return A {@link Disposable} that unregisters the provider when being disposed
-			 */
+		 * @param provider A terminal quick fix provider
+		 * @return A {@link Disposable} that unregisters the provider when being disposed
+		 */
 		export function registerTerminalQuickFixProvider(id: string, provider: TerminalQuickFixProvider): Disposable;
 	}
 
-	export class TerminalQuickFixExecuteTerminalCommand {
+	export class TerminalQuickFixTerminalCommand {
```

Comment:
I don't think this will cause breakages, need to check whether copilot references the type

Reply:
No references in `vscode-copilot` for `TerminalQuickFixExecuteTerminalCommand`